dialogue-client-verifier/src/test/resources/log4j2-test.xml

```diff
@@ -21,12 +21,6 @@
         </Console>
     </Appenders>
     <Loggers>
-        <Logger name="com.palantir.dialogue.core.ConcurrencyLimitedChannel" level="debug">
-            <AppenderRef ref="Console"/>
-        </Logger>
-        <Logger name="com.palantir.dialogue.core.AimdConcurrencyLimiter" level="debug">
-            <AppenderRef ref="Console"/>
```

Comment:
these extra console appender references result in double-logging these lines. No reason to log these at debug in our verification tests anyhow.